charters/PM/charter.md

```diff
@@ -0,0 +1,83 @@
+# PM (_Program, Product, or Project Management_) Team Charter
+
+## Description
+
+The Data Coordination Platform (DCP) PM team is responsible for the oversight of the DCP *as product*, developing the shared understanding of the overall product vision, priorities, and release schedule with the community.
+
+The core team is composed of *Project Leads* and *Product Owners* from each DCP project, *DCP UX* (Voice of the Customer), and a *Science Program Manager*.
```

Comment:
This is an aside rather than something that should stop this charter from being finalised but I want to note it here as I think of it.

At the moment we don't have a single list of all the PL, POs and TLs for these charters. It would be great if such a list could exist in a central location for easy reference. It would also be good to have some kind of pointer to the UX team as while I don't think we want a UX charter, all charters should reference user needs and how we will aim to address them, knowing who the UX "experts" are that can support or advise each individual group would be useful

Reply:
https://github.com/HumanCellAtlas/dcp-community/issues/46 Issue for this matter